<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color:#14213d;
	border:6px solid #FFDB70;
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>18</y>
       <width>185</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>A Salamat</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: #a7a7a7;
border:none;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toolbox &lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;-&lt;/span&gt; Unit Converter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Images/Exit.png</normaloff>../Images/Exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>198</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
border:2px solid #ffdb70;
background-color: rgb(255, 255, 255);
padding-left:10px;
}
QListView{
border:none;
border-radius:5px;
background-color: rgb(255, 255, 255);
margin:5px;
}
QListView::item {
padding:3px;
padding-top:6px;
padding-left:5px;
height: 8px;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUrl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border-radius:5px;
    margin: 0;
padding-left:6px;
padding-right:6px;
	color: rgb(20, 33, 61);
}
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>421</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
